src/styles/utilities/_utilities-settings.scss

```diff
@@ -0,0 +1,194 @@
+@use "sass:map";
+
+$utilities-modules: (
```

Comment:
We already merge the monday ui styles pr. I think we can delete the mock now.

Reply:
Also please pull from master before merge (you have some conflicts there :) )

Reply:
this is not the mock :) 
This is the settings that uses the mixin from styles.